plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/flex/CreateAppEngineFlexWtpProject.java

```diff
@@ -66,11 +66,30 @@
   private static final List<MavenCoordinates> PROJECT_DEPENDENCIES;
 
   static {
-    MavenCoordinates.Builder servletApi = new MavenCoordinates.Builder()
+    // matches the dependencies in javax:javaee-web-api:7.0
+    //@formatter:off
+    MavenCoordinates servletApi = new MavenCoordinates.Builder()
```

Comment:
could/should these be pulled from plugins/com.google.cloud.tools.eclipse.appengine.standard.java8/plugin.xml? If so, that can probably wait for a future PR though.

Reply:
Not in `.standard.java8` at least.